less_users/tests/test_views.py

```diff
@@ -0,0 +1,46 @@
+import pytest as pytest
+from django.urls import reverse
+
+from challenges.models import Category, Challenge
+
+
+def test_view_home(client):
+    url = reverse('home')
+    response = client.get(url)
+
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_view_challenges_unauthorised(client, challenges):
+    url = reverse('challenge_list')
+    response = client.get(url)
+
+    assert response.status_code == 200
+
+    context = response.context
+
+    # assert context['object_list'].count() == len(challenges)
+    assert set(context['object_list']) == set(challenges)
+
+    for challenge in challenges:
+        assert challenge in context['object_list']
```

Comment:
Gdzieś tam w commicie znajdziesz do tego odpowiedź :)
Plus dobrze by było jakbyś ustawiła sobie pre-commit, żeby odpalał blacka przy każdym commicie (`'` zamiast `"`).

Reply:
tak, pamiętam, że sety miałam przyrównać do siebie zamiast pętli for.

Reply:
Sety to jedna sprawa, ważniejsze jest teraz to drugie - trzymanie się formatowania podczas pisania kodu. Tutaj znowu używasz pojedynczych cudzysłowiów

Reply:
zaczynam regularnie sprawdzać kod 'black'iem - może zaradzi